src/Microsoft.Performance.SDK/Processing/Projection/ClipTimeToViewportProjection.cs

```diff
@@ -255,26 +259,29 @@ public object GetFormat(Type formatType)
                 public object Clone()
                 {
                     var result = new ClipTimeToViewportTimeRangePercentColumnGenerator<TGenerator>(
-                        ViewportSensitiveProjection.CloneIfViewportSensitive(this.timeRangeColumnGenerator.Generator));
+                        ViewportSensitiveProjection.CloneIfViewportSensitive(this.timeRangeColumnGenerator.Generator),
+                        this.viewportContainer, 
```

Comment:
VisibleTableRegionContainer is mutable (VisibleTableRegion has a setter).  Perhaps make a new instance and set its property for the cloned instance

Reply:
Resolved offline. The container needs to be shared with the cloned instance.